src/api/posts/src/storage.js

```diff
@@ -57,6 +76,20 @@ module.exports = {
 
   getFeeds: () => redis.smembers(feedsKey),
 
+  getInvalidFeeds: async () => {
+    const invalidKeys = await getFeedKeysUsingScanStream('t:feed:*:invalid');
```

Comment:
Let's not hard-code the key values, use the variables we have:

```js
`${feedNamespace}*${invalidSuffix}`
```

Reply:
Thank you, it is better than hardcode. I have rebased and updated the Pr.